src/app/business/profile/createProfile/createProfile.component.ts

```diff
@@ -229,7 +229,10 @@ export class CreateProfileComponent implements OnInit {
             isExisted:this.I18N.keyID['sds_isExisted'],
         },
         "maxIOPS": { required: this.I18N.keyID['sds_required'].replace("{0}","MaxIOPS")},
+        "minIOPS": { required: this.I18N.keyID['sds_required'].replace("{0}","MinIOPS")},
```

Comment:
What it takes to say it as "Maximum IOPS", "Minimum IOPS" etc? any specific backend support?

Reply:
Probably a coffee or two :P 
Can be done.